x/config/doc.go

```diff
@@ -24,13 +24,20 @@
 //
 // Usage
 //
-// To build a Dispatcher, set up a Configurator and inform it about the
-// different transports that it needs to support. This object is re-usable
-// and may be stored as a singleton in your application.
+// To build a Dispatcher, first create a new Configurator. This object will be
+// responsible for loading your configuration. It does not yet know about the
+// different transports, peer lists, etc. that you want to use. Inform it
+// about them by registering them with it.
```

Comment:
Change to,

> // You can inform
// the Configurator about the different transports, peer lists, etc. by
// registering them using RegisterTransport, RegisterPeerList, and
// RegisterPeerListUpdater.